src/main/java/java/time/zone/Providers.java

```diff
@@ -1,17 +0,0 @@
-package java.time.zone;
-
-import java.nio.ByteBuffer;
-import java.time.zone.ZoneRulesProvider;
-
-//JDK implementation/ real in /src/main/super
-public class Providers {
-
-	private Providers() {
-		// prevent instantiation
-	}
-
-	public static ZoneRulesProvider of(ByteBuffer data) {
-		return null;
-	}
-
-}

```

Comment:
I had to remove the Provider is transpiling to GWT was complaining about this class.